src/main/scala/fs2/kafka/KafkaAdminClient.scala

```diff
@@ -113,6 +146,18 @@ sealed abstract class KafkaAdminClient[F[_]] {
 }
 
 object KafkaAdminClient {
+  private[this] def createTopicWith[F[_]](
+    client: Client[F],
+    topic: NewTopic
+  ): F[Unit] =
+    client(_.createTopics(java.util.Collections.singleton(topic)).all.void)
```

Comment:
Would it make sense for this to instead call `createTopicsWith(client, List(topic))`, to avoid the duplicate logic?

Reply:
We could, but we avoid creating the extra intermediate `List` by doing it this way.